ansible/vars/l10k/l10k_public.yml

```diff
@@ -0,0 +1,200 @@
+SITE_HOST: 'l10k.commcarehq.org'
+COMMTRACK_SITE_HOST: 'l10k.commcarehq.org'
+NO_WWW_SITE_HOST: 'l10k.commcarehq.org'
+deploy_env: l10k
+
+active_sites:
+  cchq_ssl: True
+  cchq_redirect: True
+  cchq_http: True
+  cchq_http_j2me: False
+  cchq_http_redirect: False
+  commtrack_ssl: False
+  commtrack_http: False
+  hq_status: False
+  tableau: False
+  icds_tableau: False
+  wiki: False
+  wiki_http: False
+  motech: False
+  motech2: False
+  riakcs: False
+
+proxy_type: nginx
+
+ansible_user_password_sha_512: "{{ secrets.ANSIBLE_USER_PASSWORD_SHA_512 }}"
+fake_ssl_cert: no
+
+NEW_RELIC_KEY: "{{ secrets.NEW_RELIC_KEY }}"
+run_newrelic_plugin_agent: False
+
+DATADOG_ENABLED: True
+DATADOG_API_KEY: "{{ secrets.DATADOG_API_KEY }}"
+DATADOG_APP_KEY: "{{ secrets.DATADOG_APP_KEY }}"
+
+elasticsearch_endpoint: '{{ groups.elasticsearch.0 }}'
+elasticsearch_memory: '4096m'
+elasticsearch_cluster_name: 'deves'
```

Comment:
i don't think we should update it unless we know the side effects of changing the cluster name